src/sentry/event_manager.py

```diff
@@ -561,6 +561,8 @@ def save_error_events(
         ), metrics.timer("event_manager.calculate_event_grouping", tags=metric_tags):
             hashes = _calculate_event_grouping(project, job["event"], grouping_config)
 
+        metrics.gauge("grouping.hashes_calculated", value=2 if secondary_hashes else 1)
```

Comment:
FWIW, it will give you good measure of average hash calculations per ingest, not so great for rate or total count.

Reply:
Yeah, I was trying to figure out if it was better to use this or use `metrics.incr` with a tag for the value. I wasn't sure if the cardinality would be too high there, though. (Right now it's either `1` or `2`, but in the future it could be `1` up to `however-many-configs-we-can-ship-in-90-days`.)

Being able to give an average is nice, but I'd also like to be able to say `x%` of the time it's `1` and `y%` of the time it's `2`. Is the only way to do that with `incr`?

Reply:
I was thinking more like 
```
metrics.gauge("grouping.hashes_calculated", value=2 if secondary_hashes else 1)
if secondary_hashes:
    metrics.incr("grouping.secondary_hashes_calculated")
```

Reply:
That works for the current case, where the options are only `1` or `2`, but what about when the options become `1`, `2`, `3`, ..., `n`?

Reply:
UPDATE: Discussed off-line. TL;DR for posterity: Gauge is reported as an average, so if you hope to take an average of a gauge metric, you're feeding averages into an average, which doesn't work. `incr` it is!